include/ncasset/Assets.h

```diff
@@ -1,7 +1,11 @@
 #pragma once
```

Comment:
Since there's an update to api, version should definitely get bumped (build.yml and root CMakeLists.txt). If the engine can't directly consume this update without changes I would suggest 2.0.0.